src/services/expense.service.js

```diff
@@ -0,0 +1,62 @@
+const { Expense } = require('./../models/Expense.model');
+const { Op } = require('sequelize');
+const normalize = ({ amount, category, id, note, spentAt, title, userId }) => {
+  return {
+    amount,
+    category,
+    id,
+    note,
+    spentAt,
+    title,
+    userId,
+  };
+};
+
+const getAll = async ({ categories, from, to, ...query }) => {
+  const where = { ...query };
+
+  if (categories) {
+    where.category = categories;
+  }
+
+  if (from && to) {
+    where.spentAt = { [Op.between]: [new Date(from), new Date(to)] };
+  } else if (from) {
+    where.spentAt = { [Op.gte]: [from] };
```

Comment:
The `from` date should be wrapped in `new Date()` to ensure it's correctly interpreted as a date object, similar to how it's done in the `[Op.between]` condition.